lib/pattern_query_helper/sql.rb

```diff
@@ -2,60 +2,58 @@ module PatternQueryHelper
   class Sql
     def self.sql_query(config)
       model = config[:model]
-      query = config[:query]
       query_params = config[:query_params] || {}
       page = config[:page]
       per_page = config[:per_page]
-      filter_string = config[:filter_string]
       filter_params = config[:filter_params] || {}
-      sort_string = config[:sort_string]
 
-      if page && per_page
+      if page and per_page
```

Comment:
```suggestion
      if page && per_page
```
See https://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby.
For more details see http://www.virtuouscode.com/2010/08/02/using-and-and-or-in-ruby/